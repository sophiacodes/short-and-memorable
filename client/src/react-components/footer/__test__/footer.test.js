import React from 'react';
import Footer from '../';
import renderer from 'react-test-renderer';

it('renders <Footer /> correctly', () => {
  const tree = renderer
    .create(<Footer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});