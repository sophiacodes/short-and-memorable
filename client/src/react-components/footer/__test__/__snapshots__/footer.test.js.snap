// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Footer /> correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="app-details"
  >
    <span>
      Â© 2018
    </span>
    <span>
      Version 0.1
    </span>
    <span
      className=""
    >
      Sophia Grace Domfeh
    </span>
  </div>
  <div
    className="app-name"
  >
    <span
      className="word-count"
    >
      Word Count
    </span>
    <span
      className=""
    >
      Take home test
    </span>
  </div>
</footer>
`;
